model/src/w3profsmd_pdlib.F90

```diff
@@ -748,22 +775,25 @@ SUBROUTINE PDLIB_W3XYPUG ( ISP, FACX, FACY, DTG, VGX, VGY, LCALC )
       IP      = JSEA
       IP_glob = iplg(IP)
       ISEA    = MAPFS(1,IP_glob)
-      !write(*,*) 'IP TEST', JSEA, ISEA, IP, IP_glob
 #ifdef NOCGTABLE
       CALL WAVNU_LOCAL(SIG(IK),DW(ISEA),WN1,CG1)
       AC(IP)  = VA(ISP,JSEA) / CG1 * CLATS(ISEA)
       VLCFLX(IP) = CCOS * CG1 / CLATS(ISEA)
-      VLCFLY(IP) = CSIN * CG(IK,ISEA)
+      VLCFLY(IP) = CSIN * CG1
 #else
       AC(IP)  = VA(ISP,JSEA) / CG(IK,ISEA) * CLATS(ISEA)
       VLCFLX(IP) = CCOS * CG(IK,ISEA) / CLATS(ISEA)
       VLCFLY(IP) = CSIN * CG(IK,ISEA)
+      !WRITE(740+IAPROC,*) 'VSingle LOOP', IK, ITH, JSEA, FACX*ECOS(ITH), FACY*ESIN(ITH), CG(IK,ISEA), VLCFLX(IP), VLCFLY(IP), CLATS(ISEA)
```

Comment:
Please clean up

Reply:
I have cleaned the whole subroutine. Everything else should remain.